docs/reference/bpmn-processes/user-tasks/user-tasks.md

```diff
@@ -0,0 +1,74 @@
+---
+id: user-tasks
+title: "User tasks"
+---
+
+A user task is used to model work that needs to be done by a human actor. When
+the process execution arrives at such a user task, a new job similar to a
+[service task](../service-tasks/service-tasks) is created. The process instance
+stops at this point and waits until the task is completed.
+
+![user-task](assets/user-task.png)
+
+Applications like [Tasklist] can be used by humans to complete these tasks.
+Alternative a job worker can subscribe to the job type
+`io.camunda.zeebe:userTask` to complete the job manully.
+
+When the job is completed, the user task gets completed and the process
+instance continues.
+
+## User Task Forms
+
+User tasks supports specifing a `formKey` attribute, using the
+`zeebe:formDefinition` extension element. The form key can be used to specify
+and identifier to associate a form to the user task. [Tasklist] supports
+embedded [Camunda Forms](/product-manuals/tasklist/userguide/camunda-forms.md),
+these can be embedded into the BPMN process XML as an `zeeb:UserTaskForm`
+extension element of the process element.
+
+## Variable mappings
+
+By default, all job variables are merged into the process instance. This
+behavior can be customized by defining an output mapping at the service task.
+
+Input mappings can be used to transform the variables into a format that is
+accepted by the job worker.
+
+## Task headers
+
+A user task can define an arbitrary number of `taskHeaders`. They are static
+metadata that are handed to workers along with the job. The headers can be used
+as configuration parameters for the worker.
+
+
+## Additional resources
```

Comment:
The structure of the additional resources is a bit different from the other pages (i.e. collapsed vs always visible). I'm not against the format but we should align the sections. 

Maybe as part of #235?

Reply:
I aligned the other pages with #240